command/git_test.go

```diff
@@ -100,6 +102,146 @@ func (s *GitGetProjectSuite) TestValidateGitCommands() {
 	s.Equal(refToCompare, ref)
 }
 
+func (s *GitGetProjectSuite) TestBuildHTTPCloneCommand() {
```

Comment:
could you annotate these test cases a bit?

Reply:
done